app/build.gradle

```diff
@@ -111,6 +111,7 @@ android {
 
     lintOptions {
         disable 'InvalidPackage'
+        abortOnError false
```

Comment:
🙇  Thank you very much for your review! 

> sometimes this option prevents critical error in the build process.

I see..! I got it!

> Can we solve this by fixing multiline-collapsingtoolbar?

Ok! I will try! 👍